E/EGL/build_tarballs.jl

```diff
@@ -0,0 +1,57 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "EGL"
+version = v"1.4.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://gitlab.freedesktop.org/mesa/mesa.git", "1b74a12ea0ae900d49d1921ed8931eb6131e1f18")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+mkdir build
```

Comment:
The automatic installation of the license looks for files with reasonable names in the only directory inside `/workspace/srcdir`. If there are more than one directory here, this will fail. So it's better to enter the mesa directory and create the build folder inside it

Reply:
OK, will change that